lib/common/xmlparser.rb

```diff
@@ -684,10 +691,10 @@ def text(text_string)
               @dr_active_spells_stellar_percentage = Regexp.last_match[:percentage].to_i
               unit = Regexp.last_match[:unit]
               duration = unit == 'anlaen' ? duration * 30 : duration
-            when /(?<spell>[^<>]+?)\s+\(.+\)/i
+            when /(?<spell>^[^\(]+)\(.+\)/i
               # Spells with inexact duration verbiage, such as with
               # Barbarians without knowledge of Power Monger mastery
-              spell = Regexp.last_match[:spell]
+              spell = Regexp.last_match[:spell].strip
```

Comment:
Good catch, should line 689 also do a strip?

Reply:
That one shouldn't need a strip because in that instance, spell is always `Stellar Collector` We could even clean that up a bit, but it's not hurting anything either.